cluster/gce/config-test.sh

```diff
@@ -568,3 +568,6 @@ export GCE_PRIVATE_CLUSTER_PORTS_PER_VM=${KUBE_GCE_PRIVATE_CLUSTER_PORTS_PER_VM:
 export ETCD_LISTEN_CLIENT_IP=0.0.0.0
 
 export GCE_UPLOAD_KUBCONFIG_TO_MASTER_METADATA=true
+
+# Optoinal: Enable Windows CSI-Proxy
+ENABLE_CSI_PROXY="${ENABLE_CSI_PROXY:-true}"
```

Comment:
why is there no "export"?

Reply:
Also wondering. Why not make this look like the variables just above?